sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClient.java

```diff
@@ -464,6 +468,37 @@ TracerProvider getTracerProvider(){
         return this.tracerProvider;
     }
 
+    /**
+     * Enable throughput control by providing the throughput control groups.
+     * Each cosmos client can only enable throughput control once.
+     *
+     * @param groupList The throughput control group configuration list.
+     */
+    @Beta(value = Beta.SinceVersion.V4_12_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public void enableThroughputControl(List<ThroughputControlGroup> groupList) {
+        checkArgument(groupList != null, "Throughput control group list cannot be null");
```

Comment:
NIT - use checkNotNull precondition for cinsistency

Reply:
Changed to use CheckNotNull